server/src/test/java/org/opensearch/search/profile/query/ConcurrentQueryProfileBreakdownTests.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.search.profile.query;
+
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ConcurrentQueryProfileBreakdownTests extends OpenSearchTestCase {
+
+    public void testBuildQueryProfileBreakdownMap() {
+        Map<String, Long> testMap = new HashMap<>();
+        testMap.put("set_min_competitive_score_count", 0L);
+        testMap.put("match_count", 0L);
```

Comment:
Instead of using the raw string, could you use the pre defined String Constants

Reply:
Hi @kkmr, I have added the constants. Please help to review it again, thanks.